README.md

```diff
@@ -1757,6 +1757,27 @@ cache-dir = "~/.cache/ruff"
 
 ---
 
+#### [`comment-tags`](#comment-tags)
```

Comment:
`comment-tags` feels a bit too generic.

I guess Python almost called these `codetags` ([PEP 350](https://peps.python.org/pep-0350/)).

`todo-tags`?

Reply:
Is "codetags" perhaps the standard terminology for this?

https://marketplace.visualstudio.com/items?itemName=cg-cnu.vscode-codetags
https://trac-hacks.org/wiki/CodeTagsPlugin

Reply:
I guess in Eclipse these are "task tags" (https://stackoverflow.com/questions/9586478/ide-comment-keywords). That feels most appropriate.

Reply:
Otherwise good to merge, but curious what you think.

Reply:
Thanks for bringing this up! I agree that "comment tags" doesn't sound that great. And nice find with PEP 350 ... I didn't know about that PEP.

I am afraid there is no standard terminology for this. The PEP has been rejected and apparently only called these "codetags" because of the Trac plugin, you linked. That plugin is now also unmaintained. Sidenote ad the codetags VS Code extension: [Better Comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) is the much more commonly used extension for this, which calls these just `better-comments.tags` in the configuration and "(annotation) tags" in the documentation.

I don't like "code tag" since it's very uninformative (you cannot put these tags anywhere in the code; only in comments and the tags also don't describe the code but usually some tasks). "todo tags" is too specific since TODO is just one variant of such tags.

"task tags" is interesting. While I cannot recall hearing that term before, it does make sense and does appear to be somewhat established in academia (`task annotation` [2008](https://dl.acm.org/doi/10.1145/1368088.1368123), `task tag` [2013](https://www.cqse.eu/fileadmin/content/news/publications/2013-quality-analysis-of-source-code-comments.pdf), `task comment` [2022](https://research.ou.nl/ws/portalfiles/portal/46333009/Lung_C_IM9906_SE_AF_scriptie_Pure.pdf)).

The only reason I see against "task tags" is that sometimes such tags are also used for non-tasks e.g NOBUG (from the PEP 350) to mark well-known problems that will never be addressed due to design problems or domain limitations. However such tags are really so rare that I think it is fine to go with "task tags".

With all of this in mind "task tags" certainly seems like the best option :)